ci-operator/config/open-cluster-management/prometheus/open-cluster-management-prometheus-release-2.4.yaml

```diff
@@ -0,0 +1,69 @@
+base_images:
+  base:
+    name: ubi-minimal
+    namespace: ocp
+    tag: "8"
+  open-cluster-management_builder_go1.15-linux:
+    name: builder
+    namespace: open-cluster-management
+    tag: go1.15-linux
+  open-cluster-management_builder_go1.16-linux:
+    name: builder
+    namespace: open-cluster-management
+    tag: go1.16-linux
+build_root:
+  image_stream_tag:
+    name: builder
+    namespace: open-cluster-management
+    tag: go1.16-linux
+images:
+- dockerfile_path: Dockerfile.prow
+  from: base
+  inputs:
+    open-cluster-management_builder_go1.16-linux:
+      as:
+      - registry.ci.openshift.org/open-cluster-management/builder:go1.16-linux
+  to: prometheus
+promotion:
+  name: "2.4"
+  namespace: open-cluster-management
+resources:
+  '*':
+    limits:
+      memory: 6Gi
+    requests:
+      cpu: 100m
+      memory: 1Gi
+tag_specification:
+  name: "4.7"
+  namespace: ocp
+test_binary_build_commands: "true"
+tests:
+- as: vendor
+  commands: make unused && git diff --exit-code
+  container:
+    from: src
+- as: common-test
+  commands: GOOPTS="-tags=builtinassets -p=1" make common-test
+  container:
+    from: src
+- as: react-app-test
+  commands: |
+    nodejs_version=v14.16.1
+    nodejs_platform=linux-x64
+    nodejs="node-${nodejs_version}-${nodejs_platform}"
+    cd /tmp
+    curl -LO "https://nodejs.org/download/release/${nodejs_version}/${nodejs}.tar.gz"
+    tar xf "${nodejs}.tar.gz"
+    export NODEJS_HOME="/tmp/${nodejs}"
+    export PATH=$PATH:$NODEJS_HOME/bin
+    HOME=/tmp npm install yarn
+    export PATH=$PATH:/tmp/node_modules/.bin
+    cd -
+    make react-app-test
+  container:
+    from: src
```

Comment:
If you use the nodejs builder image, you don't need to install node or yarn.

Reply:
thanks, we need to build golang img.